ui/component/claimPreview/view.jsx

```diff
@@ -223,7 +223,7 @@ const ClaimPreview = forwardRef<any, {}>((props: Props, ref: any) => {
 
   const handleNavLinkClick = (e) => {
     if (onClick) {
-      onClick(e, indexInContainer);
+      onClick(e, claim, indexInContainer); // not sure indexInContainer is used for anything.
```

Comment:
The index was previously used in `handleRecommendationClicked`, but seems like the index is determined differently now.
Can retain for other future clients, or remove entirely (I can clean up later)